packages/serverless-nextjs-component/examples/multiple-instance-environment/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "multiple-instance-environment",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "deployStaging": "serverless --stage=staging",
+    "deployProd": "serverless --stage=prod"
+  },
+  "dependencies": {
+    "next": "9.1.1",
+    "react": "16.11.0",
+    "react-dom": "16.11.0"
+  },
+  "devDependencies": {
+    "dotenv": "^8.1.0",
+    "@serverless/core": "^1.1.2",
```

Comment:
You might consider adding `"serverless-next.js": "latest" so people can run this right away.  It looks like serverless-next.js loads `@serverless/core` so that could be removed but `@serverless/template` and `serverless` are needed.

Reply:
Hi,

I have not set `"serverless-next.js"` to latest since I don't want to break with the other examples that do not do this (it also allows for one to debug/test things easier). And while I agree I could probably remove `@serverless/core` (although I have not tested this), I want it in here to make this as stand alone as possible. You see, this recipe is not well known, and I suspect that other people will take it and adapt it to their needs (possibly without `"serverless-next.js"`)